templates/macros/docs.html

```diff
@@ -100,3 +103,25 @@
     {% endif %}
   </ul>
 {% endmacro %}
+{% macro on_this_page(toc) %}
+<nav aria-label="On This Page" class="on-this-page media-content">
```

Comment:
I think `on-this-page` should define it's own styling for `<a>` elements and not rely on `media-content` for this.

Reply:
done